helm/cluster-api/templates/hook-delete-aggregated-roles/job.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - |
           kubectl delete clusterrole capi-kubeadm-control-plane-aggregated-manager-role --ignore-not-found
           kubectl delete clusterrole capi-aggregated-manager-role --ignore-not-found
-        image: '{{ .Values.images.domain }}/giantswarm/docker-kubectl:1.23.1'
```

Comment:
Should we bump the version here to match?

Reply:
done